aospy/test/test_automate.py

```diff
@@ -351,3 +354,25 @@ def test_permute_aux_specs(self, calc_suite, calcsuite_specs):
         assert len(actual) == len(expected)
         for act in actual:
             assert act in expected
+
+
+@pytest.mark.parametrize(
+    ('var'),
```

Comment:
Here and elsewhere: parentheses around a single object do nothing: `('var') == 'var'`.  So I would omit them, except where they are used to break expressions over multiple lines.

Conversely, if there is a trailing comma, it defines a length-1 tuple: e.g. `('var',)`.